test/testE2E.t.sol

```diff
@@ -37,6 +40,8 @@ contract E2ETests is Test {
     address fakeDisputedReporter = bob;
     address fakeDisputeInitiator = alice;
     uint256 fakeSlashAmount = 50;
+    uint32 public fakeWeightToFee = 5000;
+    uint8 public fakeDecimals = 10;
```

Comment:
Minor thing, but can you set this to 12 as that is what is used in the pallet and the parachain node template: https://github.com/substrate-developer-hub/substrate-parachain-template/blob/main/node/src/chain_spec.rs#L68

Reply:
Still needs addressing...